src/cli/domain/get-components-by-dir.js

```diff
@@ -4,47 +4,41 @@ var fs = require('fs-extra');
 var path = require('path');
 var _ = require('underscore');
 
-module.exports = function(dependencies){
-  var logger = dependencies.logger;
-  return function(componentsDir, callback){
-
-    try {
-      var components = fs.readdirSync(componentsDir).filter(function(file){
-
-        var filePath = path.resolve(componentsDir, file),
-            isDir = fs.lstatSync(filePath).isDirectory(),
-            packagePath = path.join(filePath, 'package.json');
+module.exports = function(){
 
-        if(!isDir || !fs.existsSync(packagePath)){
-          return false;
-        }
+  return function(componentsDir, callback){
 
-        var content;
+    var isOcComponent = function(file){
 
-        try {
-          content = fs.readJsonSync(packagePath);
-        }
-        catch(err)
-        {
-          logger.log(('error reading ' + packagePath + ' ' + err.toString()).red);
-          return false;
-        }
+      var filePath = path.resolve(componentsDir, file),
+          packagePath = path.join(filePath, 'package.json'),
+          content;
 
-        if(!content.oc || !!content.oc.packaged){
-          return false;
-        }
+      try {
+        content = fs.readJsonSync(packagePath);
```

Comment:
Previous behaviour was to check dir existed, check package.json existed, and then trying to read. For simplicity, let's just try to read the Json in that folder. If it's a component, it will be there. If it's not, let's skip the dir.